cupy/cudnn.py

```diff
@@ -326,3 +330,53 @@ def add_tensor(handle, alpha, biasDesc, biasData, beta, srcDestDesc,
                srcDestData):
     cudnn.addTensor_v3(handle, alpha, biasDesc,
                        biasData, beta, srcDestDesc, srcDestData)
+
+
+class DropoutStates(object):
+
+    def __init__(self, handle, seed):
+        state_size = cudnn.dropoutGetStatesSize(handle)
+        self.states = cupy.empty((state_size,), dtype='b')
```

Comment:
How about changing name to `self._states`?
And, How about using `cupy.cuda.malloc` directory, because user don't edit `self.states`.
Python object generation cost is very high.